tests/periph_timer/main.c

```diff
@@ -57,7 +58,7 @@ static int test_timer(unsigned num)
     }
 
     /* initialize and halt timer */
-    if (timer_init(TIMER_DEV(num), TIM_SPEED, cb) < 0) {
+    if (timer_init(TIMER_DEV(num), TIM_SPEED, cb, NULL) < 0) {
```

Comment:
What do you think about actually using `arg` for demonstration resons in this test?